test/code/plugins/patch_management_lib_test.rb

```diff
@@ -461,4 +461,23 @@ def test_filter_updaterun_progress
     assert_equal(fakeStartDate, result['DataItems'][0]["StartTime"])
   end
   
+    def test_filter_yum_updaterun_progress
+    record =
+        {
+            'update-action-date' => 'Nov 03 20:20:30',
+            'update-action' => 'updated',
+            'package-name' => 'libXft-2.3.2-1.el6.x86$-1.28-12.el6.x86_64'
+        }
+
+    @linuxUpdatesInstance.expects(:getUpdateRunName).returns(@fakeUpdateRunName)
+
+    result = @linuxUpdatesInstance.process_yum_update_run(record, 'tag', 'HostName', Time.now)
+    assert(result != nil)
+    assert_equal(1, result['DataItems'].size)
+    assert_equal("HostName", result['DataItems'][0]["Computer"])
+    assert_equal(@fakeUpdateRunName, result['DataItems'][0]["UpdateRunName"])
+    assert_equal('libXft-2.3.2-1.el6.x86$-1.28-12.el6.x86_64', result['DataItems'][0]["UpdateTitle"])
+    assert_equal('Succeeded', result['DataItems'][0]["Status"])
```

Comment:
please add more test cases and checks:
  update-action: Installed, Erased or otherwise
  Start-Date check